build_defs/python.build_defs

```diff
@@ -196,6 +202,21 @@ def python_binary(name:str, main:str, srcs:list=[], resources:list=[], out:str=N
     )
 
 
+def _set_shebang(interpreter:str, interpreter_options:str, site:bool):
+    """Returns a shebang to pass to arcat to inject at the top of a pex"""
+    # Assume the path to the interpreter is relative
+    shebang = f'/usr/bin/env {interpreter}'
+    if not site:
+        if ' ' in shebang:
```

Comment:
I'm not sure i understand what's going on here. We just set shebang above so this is always going to be true?

Reply:
Trying to remember what was going on here, but you're right that does look weird. Think I've fixed it.